front/tests/utils/manchette.ts

```diff
@@ -13,31 +13,36 @@ export type Waypoint = {
   checked?: boolean;
 };
 
-const firstRequestedPointKey = `${frScenarioTranslations.requestedPointUnknown} (1)`;
-const secondRequestedPointKey = `${frScenarioTranslations.requestedPointUnknown} (2)`;
+const requestedDestination = frScenarioTranslations.requestedDestination;
+
+function requestedPoint(number: string): string {
+  const requestedPointFrame = frScenarioTranslations.requestedPoint;
+  const count = new RegExp('{{ count }}', 'gi');
+  return requestedPointFrame.replaceAll(count, number);
+}
```

Comment:
I don't really see the need of the `RegExp`
```suggestion
function requestedPoint(number: string): string {
  return frScenarioTranslations.requestedPoint.replace('{{ count }}', number);
}
```

Reply:
🙏 Much better indeed. [Done](https://github.com/OpenRailAssociation/osrd/pull/13521/commits/05a58f08d81479b40bafbc6696ff6d22cdeb0115).
Now I resolve this, then rebase+merge when ready.